v3.3/glfw/GLFW_C_REVISION.txt

```diff
@@ -0,0 +1 @@
+e4e9581557826bd522158ab9159859e20e10700e
```

Comment:
I note that this choice of revision is after the `3.3` tag.

Is that really what we want? Or do we want to stick to "released" versions? I could be persuaded either way.

Reply:
On the [glfw.org/download](https://www.glfw.org/download.html) webpage, it is stated:
>  the 3.3-stable branch only adds bug fixes for patch releases.

I'm guessing we want the latest patch available in `go-gl/glfw`